townhall/myapi/models.py

```diff
@@ -77,3 +77,22 @@ class Comment(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+class Task(models.Model):
+
+    STATUS_CHOICES = [
+        ('open', 'Open'),
+        ('in_progress', 'In Progress'),
+        ('completed', 'Completed'),
+    ]
+
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    deadline = models.DateTimeField()
+    status = models.CharField(max_length=20)
```

Comment:
status can be defined using the choices as seen here https://docs.djangoproject.com/en/5.1/ref/models/fields/#enumeration-types

Reply:
I made the modification